### PR TITLE
Cookie accepted

### PR DESCRIPTION
**Pull request by Dann1y**
스크린샷 2021-03-07 오후 4 26 52

Published CookieAlert Component

### CODE REVIEW

**Comment by softmarshmallow** [comment]
@Dann1y notify @GwonHeeJun when PR is ready to be accepted

File: web/components/cookie-accept/index.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  width: 145px;
+  height: 35px;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  margin-left: auto;
+`;
+
+const Button = styled.div<Color>`
+  font-family: Roboto;
```
Comment (GwonHeeJun) [inline web/components/cookie-accept/index.tsx]: font style is global style, didn't use that

**Review by GwonHeeJun — COMMENTED** [review]
Plz check publish style review

File: web/components/cookie-accept/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 17px;
+
+  color: #4e4e4e;
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
```
[the remark this answers is not in the document]
Comment (Dann1y) [reply web/components/cookie-accept/index.tsx]: I thought I had to add these.
These were in the Figma Inspect.

File: web/components/cookie-accept/index.tsx (hunk 1 above)
Comment (Dann1y) [reply web/components/cookie-accept/index.tsx]: Okay. I'm on it

File: web/components/cookie-accept/index.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  background-color: #ffffff;
+  z-index: 998;
+
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`;
+
+const Wrapper = styled(Flex)`
+  height: 107;
```
[the remark this answers is not in the document]
Comment (Dann1y) [reply web/components/cookie-accept/index.tsx]: Oops

File: web/components/cookie-accept/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (GwonHeeJun) [reply web/components/cookie-accept/index.tsx]: plz remove figma inspect value ( glow... )

**Comment by GwonHeeJun** [comment]
Cookie accpect screen hasn't diff?

**Comment by GwonHeeJun** [comment]
Add, Plz use SectionLayout, ( variant : "content-default" )

File: web/components/cookie-accept/index.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Flex } from "rebass";
+
+const CookieAccept: React.FC = () => {
+  return (
+    <Positioner>
+      <Wrapper width={["100%", "728px", "984px", "1040px"]} px="20px">
```
Comment (GwonHeeJun) [inline web/components/cookie-accept/index.tsx]: Wrapper To SectionLayout, 

reference ( /section/bridged/[sectionName] )

**Comment by softmarshmallow** [comment]
Saving "Cookie accepted"
Options on table

- https://dev.to/debosthefirst/how-to-use-cookies-for-persisting-users-in-nextjs-4617
- https://github.com/maticzav/nookies

How'd you guys like to implement

**Pull request merged**